rllib/tuned_examples/ppo/cartpole-truncated-ppo.py

```diff
@@ -18,6 +18,7 @@
         evaluation_interval=1,
         evaluation_num_workers=1,
     )
+    .framework(eager_tracing=True)
```

Comment:
On the new stack, tf2 is currently quite slow. Learning behaviour is the same though.
Eager tracing helps with this.